src/condition.cpp

```diff
@@ -835,6 +836,15 @@ conditional_t::func f_is_wearing( const JsonObject &jo, std::string_view member,
     };
 }
 
+conditional_t::func f_in_world( const JsonObject &jo, std::string_view member)
+{
+    str_or_var world_prefix = get_str_or_var( jo.get_member( member ), member, true );
+    //so the compiler doesn't complain about this var being unused
```

Comment:
What var was it complaining about?

Reply:
iirc it was "is_npc" but I couldn't get the compiler to shut up about it so i removed it entirely. Forgot about the comment